src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -185,7 +185,7 @@ internal class CommandLineParameterParser
             "file",
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-macos`
ConsoleHost unit tests.-Login pwsh switch.Doesn't falsely recognise -Login when used after -Command
```powershell
Expected strings to be the same, but they were different.
String lengths are both 6.
Strings differ at index 1.
Expected: '-login'
But was:  '-Login'
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 285
285:             $result | Should -BeExactly '-login'
```